package.json

```diff
@@ -62,6 +62,7 @@
     "ember-cli-sri": "^2.1.0",
     "ember-cli-template-lint": "^0.7.5",
     "ember-cli-uglify": "^1.2.0",
+    "ember-click-outside": "^0.1.12",
```

Comment:
saw this in ember-osf-web, allows popover to work more like a dropdown where clicking outside makes it go away